package/yast2-network.spec

```diff
@@ -23,13 +23,10 @@ Release:        0
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-Group:          System/YaST
-License:        GPL-2.0
 # should be required by devtools
-BuildRequires:  perl-XML-Writer pkgconfig rpm
```

Comment:
Sorry, now you're mixing up formatting changes done by `/usr/lib/obs/service/format_spec_file.files/prepare_spec` with semantic changes to the dependencies. I do not care if we do either of them, but I do care to not mix them in a single commit.